s3/pkg/datastore/common/datatypes.go

```diff
@@ -0,0 +1,17 @@
+package common
+
+type PutResult struct {
+	// bytes written to backend.
+	Written int64
```

Comment:
please rename the field...to BytesWritten or similar

Reply:
Good suggestion, i changed the name from Written to BytesWritten.